styleguide/source/_patterns/03-organisms/article-teaser.twig

```diff
@@ -0,0 +1,37 @@
+{% set teaserSize = teaser.size ? "joe__article-teaser--" ~ teaser.size : "" %}
```

Comment:
@aCyborg since this only contains atoms and variables and not other molecules it could be set as a molecule instead of an organism.